src/annotator/config/index.js

```diff
@@ -53,67 +61,104 @@ function configurationKeys(appContext) {
       return contexts.notebook;
     case 'all':
       // Complete list of configuration keys used for testing.
-      return [...contexts.annotator, ...contexts.sidebar, ...contexts.notebook];
+      return [
+        ...contexts.annotator,
+        ...contexts.sidebar,
+        ...contexts.notebook,
+        // "experimental" currently has no uses in any app contexts, but is included
+        // for test coverage
```

Comment:
If the `experimental` field is not actively used, and isn't mentioned in the [public documentation](https://h.readthedocs.io/projects/client/en/latest/publishers/config/) is there anything stopping us from just removing it?

Reply:
Negative, lets remove it.